src/quiz/quiz.service.ts

```diff
@@ -1,12 +1,16 @@
-import { Injectable } from "@nestjs/common";
+import { Injectable } from '@nestjs/common';
+import { CreateQuizDto } from './dto/create-quiz.dto';
 
 @Injectable()
 export class QuizService {
   getQuiz(): string {
```

Comment:
i think would be better  getQuizes instead getQuiz  and  getQuiz or getQuizById  instead  getOneQuiz.   getOneQuiz seems like get random quiz.

Reply:
Good point. I didn't develop get requests, just fixed typos. So I wasn't sure that I could change their names. But now they are renamed.